sql/sigs/pgrouting--3.6.sig

```diff
@@ -188,8 +188,13 @@ pgr_kruskaldfs(text,anyarray,bigint)
 pgr_kruskaldfs(text,bigint,bigint)
 pgr_kruskal(text)
 _pgr_kruskal(text,anyarray,text,bigint,double precision)
-_pgr_ksp(text,bigint,bigint,integer,boolean,boolean)
+_pgr_ksp(text,anyarray,anyarray,integer,boolean,boolean)
+pgr_ksp(text,anyarray,anyarray,integer,boolean,boolean)
+pgr_ksp(text,anyarray,bigint,integer,boolean,boolean)
```

Comment:
You are overdoing it, with just 2 functions is enough:
See:
https://github.com/pgRouting/GSoC-pgRouting/blob/main/sql/dijkstra/_dijkstra.sql#L38
https://github.com/pgRouting/GSoC-pgRouting/blob/main/sql/dijkstra/_dijkstra.sql#L63
The one on L28 is used on one to one one to many, many to one and many to many, the one in line 63 is used on combinations
Also, about the function that existed previously that you have to keep, see how it is handled:
https://github.com/pgRouting/GSoC-pgRouting/blob/main/sql/dijkstra/_dijkstra.sql#L91

Reply:
hmmm, I got mixed up, I placed this comment on the wrong location
anyway, just make sure you only add only two `_pgr_ksp` functions.

Reply:
and delete none